### PR TITLE
chore(license/contributing): Added license file and contributing file

### PR DESCRIPTION
**Pull request by peterdolan**
JIRA: ENG-894

What: Added license and contributing files. Supremely simple change -- virtually copied and pasted from node version with relevant links changed. Also made one stylistic change on the wrappers, merely adding an extra line.
Why: Generating consistency amongst all the wrappers we have!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# How to contribute
+We welcome contributions from the community and are pleased to have them.  Please follow this guide when logging issues or making code changes.
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-python/issues).  Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the earliest version that has the issue being reported.
```
Comment (sjl2) [inline CONTRIBUTING.md]: Thoughts on having this link to [here](https://github.com/lob/lob-python/issues/new) instead?

Though, I think I may be fore keeping it where it, but also adding a line to make sure they check the closed issues to ensure that it isn't a solved or known problem.

**Comment by sjl2** [comment]
LGTM pending one comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (peterdolan) [inline CONTRIBUTING.md]: yup I agree

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-python/issues/new).  Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the earliest version that has the issue being reported.
+
+## Patching Code
+Code changes are welcome and should follow the guidelines below.
+
+* Fork the repository on GitHub.
+* Add tests for your new code ensuring that you have 100% code coverage (we can help you reach 100% but will not merge without it).
+    * Run `npm test` to generate a report of the test coverage
```
Comment (dmlittle) [inline CONTRIBUTING.md]: Can you fix the command to be the python tests. Still has the node commands.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* [Pull requests](https://help.github.com/articles/about-pull-requests/) should be made to the [master branch](https://github.com/lob/lob-python/tree/master).
+
+To help encourage consistent code style guidelines, we have included an `.editorconfig` file for use with your favorite [editor](http://editorconfig.org/#download).
```
Comment (dmlittle) [inline CONTRIBUTING.md]: We don't have an `.editorconfig` for this repo. Can you add one?

**Comment by dmlittle** [comment]
Can you drop the changes to the LICENSE file name? The standard is to have it be just named LICENSE. We could rename the lob-node one, but it's a minor change so not really necessary. Also, can you update the year in the LICENSE to be 2013-2016.

File: CONTRIBUTING.md (hunk 2 above)
Comment (peterdolan) [inline CONTRIBUTING.md]: good catch

File: LICENSE [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
```
Comment (peterdolan) [inline LICENSE]: The above is the same -- idk why github is separating it this weird way

File: LICENSE.txt [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
```
Comment (peterdolan) [inline LICENSE.txt]: this file also definitely doesn't exist anymore

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-python/issues/new).  Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the earliest version that has the issue being reported.
+
+## Patching Code
+Code changes are welcome and should follow the guidelines below.
+
+* Fork the repository on GitHub.
+* Add tests for your new code ensuring that you have 100% code coverage (we can help you reach 100% but will not merge without it).
+    * Run nosetests to generate a report of the test coverage
```
Comment (dmlittle) [inline CONTRIBUTING.md]: can you change it to `nosetests` with  `` ` `` so it's highlighted

File: LICENSE (hunk 4 above)
Comment (dmlittle) [inline LICENSE]: The line length is different so the wrapping is different. You can fix it but it's not necessarily a big deal as long as the text is the same.

**Comment by dmlittle** [comment]
LGTM. Can you update the main PR description to be up to date for good record keep sake?

**Pull request merged**